src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -39,12 +39,12 @@ public static final class Drivetrain {
         public static final double driveGearing = 6.75;
 
         public static final double driveModifier = 1;
-        public static final double wheelDiameterMeters = Units.inchesToMeters(4.0) * 7.36/7.65 /* empirical correction */;
+        public static final double wheelDiameterMeters = Units.inchesToMeters(4.0);
```

Comment:
The empirical correction you removed might be important for accurate auto odometry.

Reply:
Fair, I'll undo the change.